child_compassion/models/child_lifecycle_event.py

```diff
@@ -4,13 +4,16 @@
 #    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
 #    Releasing children from poverty in Jesus' name
 #    @author: Emanuel Cino <ecino@compassion.ch>
+#             Philippe Heer <heerphilippe@msn.com>
```

Comment:
You can add your name after mine at line 6 (and remove e-mails if line is too long)